addons/vscode/CHANGELOG.md

```diff
@@ -207,3 +207,11 @@ Add preview button
 
 - Bug fix:
   - Scroll to cursor on 2-column documents is now improved.
+
+## v0.10.0
+
+- Features:
+  - Bump to typst v0.10.0
+
+- Known issues:
+  - Pattern fill is temporarily unavailable. This will be fixed in the next release.
```

Comment:
pls check this @Myriad-Dreamin

Reply:
it is already supported, removing this line

Reply:
I did it yesterday.